doc-content/enterprise-only/installation/clustering-smart-router-install-proc.adoc

```diff
@@ -71,16 +69,17 @@ KieServicesConfiguration config = KieServicesFactory.newRestConfiguration("http:
 +
 In this example, `smartrouter.example.com` is the Smart Router URL, and `USERNAME` and `PASSWORD` are the log in credentials for the Smart Router configuration.
 
-[NOTE]
-====
-You must create containers directly against the `kie-server`. For example:
-
+. To create a new container in an umanaged {KIE_SERVER} so that you can fill it with example data, send the following HTTP request:
++
+[source]
 ----
 $ curl -v -X POST -H 'Content-type: application/xml' -H 'X-KIE-Content-Type: xstream' -d @create-container.xml -u ${KIE_CRED} http://${KIE-SERVER-HOST}:${KIE-SERVER-PORT}/kie-server/services/rest/server/config/
 ----
-
++
+. Review the content the `create-container.xml` file:
```

Comment:
[Typo] Add "of" and pluralize "contents", or simplify:

s/Review the content the `create-container.xml` file:/Review the contents of the `create-container.xml` file:/Review the `create-container.xml` file:

Reply:
Good catch.